Sources/TuistGraph/Models/Settings.swift

```diff
@@ -99,12 +100,6 @@ public struct Settings: Equatable, Codable {
             defaultSettings: defaultSettings
         )
     }
-
-    // MARK: - Equatable
-
-    public static func == (lhs: Settings, rhs: Settings) -> Bool {
-        lhs.base == rhs.base && lhs.configurations == rhs.configurations
-    }
```

Comment:
Was there a reason why this was ignoring `defaultSettings`? @pepicrft @fortmarek

Reply:
not that I know of, I wouldn't worry about it.